manim/utils/space_ops.py

```diff
@@ -100,21 +131,67 @@ def quaternion_from_angle_axis(angle, axis, axis_normalized=False):
         return np.append(np.cos(angle / 2), np.sin(angle / 2) * normalize(axis))
 
 
-def angle_axis_from_quaternion(quaternion):
+def angle_axis_from_quaternion(quaternion: np.ndarray) -> float:
```

Comment:
Isn't this a tuple of floats?

Reply:
https://stackoverflow.com/questions/40181344/how-to-annotate-types-of-multiple-return-values As mentioned, it returns a tuple. Please don't resolve conversations that have not been fixed, it makes it hard to keep track of changes.